src/index.html

```diff
@@ -11,7 +11,119 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input type="text">
+        </label>
+        <label class="label">
+          Name:
+          <input type="text" maxlength="10" minlength="3">
+        </label>
+        <label class="label">
+          How old are You?
+          <input
+            type="number"
+            step="3"
+            value="12"
+            min="1"
+            max="100">
```

Comment:
same with closing bracket
```suggestion
            max="100"
          >
```